data/applications/pachyderm.yaml

```diff
@@ -6,8 +6,8 @@ spec:
   displayName: Pachyderm
   provider: Pachyderm
   description: |-
-    An enterprise-grade, open source data science platform that makes explainable, repeatable, and
-    scalable ML/AI a reality.
+    Pachyderm is the data foundation for machine learning.  
+    We provide industry-leading data versioning, pipelines, and lineage that allow data science teams to automate the machine learning lifecycle and optimize their machine learning operations (MLOps). 
```

Comment:
We try to keep the description text to 4 lines in the tile if at all possible. Any way to reduce this?

![image](https://user-images.githubusercontent.com/11633780/134177182-efbb5488-f5e1-4fce-b381-68c509275855.png)